.npmrc

```diff
@@ -1 +1 @@
-use-node-version=16.19.0
+registry=https://registry.npmjs.org/
```

Comment:
Remove this, I needed this on my machine, because it was attempting to fetch from artifactory for some reason.